docs/api/menu-item.md

```diff
@@ -108,7 +108,7 @@ The following additional roles are available on _macOS_:
 * `moveTabToNewWindow` - Map to the `moveTabToNewWindow` action.
 * `window` - The submenu is a "Window" menu.
 * `help` - The submenu is a "Help" menu.
-* `services` - The submenu is a "Services" menu.
+* `services` - The submenu is a ["Services"](https://developer.apple.com/documentation/appkit/nsapplication/1428608-servicesmenu?language=objc) menu; this is not the same as a menu item populated with [NSServices](https://developer.apple.com/documentation/bundleresources/information_property_list/nsservices?language=objc) and should not be used in context menus.
```

Comment:
> a menu item populated with [NSServices]

I'm not sure this is a correct description. The "Services" submenu in a context menu is not populated with our own services (as defined by the `NSServices` plist key), but rather with services from other apps – just like the Application menu's services menu, but with a different layout and less entries.

Assuming the short term solution is indeed just to change the documentation, how about:

```suggestion
* `services` - The submenu is a ["Services"](https://developer.apple.com/documentation/appkit/nsapplication/1428608-servicesmenu?language=objc) menu. This is only intended for use in the Application Menu and is *not* the same as the "Services" submenu menu used in context menus in macOS apps, which is not implemented in Electron.
```

Reply:
thanks ⭐️